feature/office/src/main/java/com/iteneum/office/presentation/ui/OfficeViewModel.kt

```diff
@@ -65,8 +66,8 @@ class OfficeViewModel() : ViewModel() {
  */
 val officeResponse: DataState<Office> = DataState.Success(
     Office(
-        "4950 Gaidrew, Alpharetta, GA, 30022 ",
-        "9AM-6PM",
+        "4950 Gaidrew, Alpharetta, GA, 30022",
+        "Hours: 9AM-6PM",
```

Comment:
Replace **Hours: 9AM-6PM** to **9AM-6PM** without **Hours**
Aditionally add named params for **address** and **schedule**